README.adoc

```diff
@@ -16,21 +16,9 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
 
 == Development
 
-To run this operator locally you need to have at least one Minishift profile started:
-
-```bash
-$ minishift start --profile host
-```
-
-Then you can run the operator locally with the help of `operator-sdk` (you need version v0.19.2):
-
-```bash
-$ make up-local
-```
 
 That Makefile target takes care of additional several steps which can be executed separately:
```

Comment:
Same comment as in the member-operator repo. Seems like this sentence is referring to the make target up-local that we're removing above. Maybe it's better to point people to the e2e repo for development? They can use the dev- targets in the e2e repo. Not sure what we want to do with the bullet points below, maybe keep them for informational purposes only?

Reply:
@rajivnathan Thanks, that makes sense. Added a line saying toolchain-e2e can be used for starting instead, lemme know if you think of better wordings.